src/penn_chime/presentation.py

```diff
@@ -268,7 +276,12 @@ def display_sidebar(st, d: Constants) -> Parameters:
 
     st.sidebar.markdown("### Spread and Contact Parameters [ℹ]({docs_url}/what-is-chime/parameters)"
                         .format(docs_url=DOCS_URL))
-    doubling_time = doubling_time_input()
+    if st.sidebar.checkbox("I know the number of days since first hospitalized case"):
```

Comment:
Please make this "I know the admission date of the first hospitalized case", so that #146 is easier, and so users don't have to repeat date arithmetic with every use, or accidentally forget to update the number on their sticky note, whiteboard, config file, or whatever.

Reply:
that's a good idea! saw your comment in the issue. i'd like to try to squeeze it in this PR! gonna work on bigger problems i'm having first.

Reply:
covered this.